lib/init.php

```diff
@@ -57,3 +61,45 @@ function gutenberg_menu() {
 	);
 }
 add_action( 'admin_menu', 'gutenberg_menu', 9 );
+
+if ( ! function_exists( 'gutenberg_render_blocks_from_request' ) ) {
+	/**
+	 * Render blocks from a REST API request.
+	 *
+	 * @param mixed $request The current request.
+	 *
+	 * @return string
+	 */
+	function gutenberg_render_blocks_from_request( $request ) {
+		$blocks = $request->get_json_params();
```

Comment:
This does not work because the shape of blocks coming over is:

```
[
  {
    "clientId": "1d788c8f-9347-4bc5-9be2-54e4298282f2",
    "name": "core/template-part",
    "isValid": true,
    "attributes": {
      "slug": "header",
      "theme": "twentytwentyfour",
      "tagName": "header",
      "area": "header"
    },
    "innerBlocks": [],
    "originalContent": "",
    "validationIssues": []
  }, ...
```